ariadne/executable_schema.py

```diff
@@ -22,23 +26,42 @@ def make_executable_schema(
 
     ast_document = parse(type_defs)
     schema = build_ast_schema(ast_document)
+    cleaned_bindables: List[SchemaBindable] = clean_bindables(*bindables)
```

Comment:
Could be inlined in `for bindable in clean_bindables(*bindables):`

Reply:
I would agree, but please notice, that, `clean_binables` (now `flat_binables`) are also passed in `repair_default_enum_values`. That is the reason of extraction of that variable.